packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -1258,6 +1258,14 @@ module('Integration | operator-mode', function (hooks) {
     await waitFor(`[data-test-stack-card-index="2"]`);
     assert.dom(`[data-test-stack-card-index="2"]`).exists();
     await click('[data-test-stack-card-index="0"] [data-test-card-header]');
+
+    await waitUntil(
+      () => !document.querySelector('[data-test-stack-card-index="2"]'),
+    );
+    await waitUntil(
+      () => !document.querySelector('[data-test-stack-card-index="1"]'),
+    );
```

Comment:
Added `waitUntil` in the test to make sure `dismissStackedCardsAbove` task (not animation) completes. There are several `await` in the task so I suspect the test was expected immediate response after dismiss.

Reply:
There is a better way to write this using waitFor. 

```
await waitFor(‘[data-test-stack-card-index=“2”]’, { count: 0 });
await waitFor(‘[data-test-stack-card-index=“1”]’, { count: 0 });
```

Reply:
I wonder tho is we are missing test waiters that could allow us not to have ti write the above tho… are we sure that if we take out the animation this wait still needs to happen?

Reply:
Yeah @habdelra , I tried removing the all the animation classes during test. It will fail if we exclude the `waitFor`